src/test/HostActivationTests/ComponentDependencyResolution.cs

```diff
@@ -447,38 +447,36 @@ public void MultiThreadedComponentDependencyResolutionWhichFailures()
 
         public class SharedTestState : IDisposable
         {
-            public TestProjectFixture PreviouslyPublishedAndRestoredPortableApiTestProjectFixture { get; set; }
-            public TestProjectFixture PreviouslyPublishedAndRestoredComponentWithNoDependenciesFixture { get; set; }
-            public TestProjectFixture PreviouslyPublishedAndRestoredComponentWithDependenciesFixture { get; set; }
-            public TestProjectFixture PreviouslyPublishedAndRestoredComponentWithResourcesFixture { get; set; }
-            public RepoDirectoriesProvider RepoDirectories { get; set; }
-
-            public string BreadcrumbLocation { get; set; }
+            public TestProjectFixture HostApiInvokerAppFixture_Built { get; }
+            public TestProjectFixture ComponentWithNoDependenciesFixture_Published { get; }
+            public TestProjectFixture ComponentWithDependenciesFixture_Published { get; }
+            public TestProjectFixture ComponentWithResourcesFixture_Published { get; }
+            public RepoDirectoriesProvider RepoDirectories { get; }
```

Comment:
I would remove the `_Published` and `_Built` suffixes. They add little to no value. The tests need specific setup, part of which is for these fixtures to be "prepped" correctly. I don't see a need to describe the preparation in the property name. Especially since there's no other variant of the fixture anyway.

Reply:
Applies to other source files as well...

Reply:
Removed (except for in some of the activation tests which explicitly test built vs published output)